src/mock-doc/element.ts

```diff
@@ -297,6 +314,25 @@ export class MockSVGElement extends MockElement {
   getTotalLength(): number {
     return 0;
   }
+  getComputedTextLength(): number {
+    return 0;
+  }
+  createSVGPoint() {
```

Comment:
This method appears to not exist on `SVGElement`, but [rather on `SVGPoint`](https://www.w3.org/TR/SVG11/coords.html#InterfaceSVGPoint). Can we please move these to a (new) `SVGTextContentElement` class that extends (a new) `MockSSVGPoint`?  The reason being is that these methods shouldn't be available on MockSVGElement (and by extension, any of its child classes)

Reply:
It looks like `createSVGPoint()` exists in [SVGSVGElement](https://www.w3.org/TR/SVG/struct.html#InterfaceSVGSVGElement) so I created that instead